### DISCUSSION TITLE
Table 22, Panel 1

### DISCUSSION THREAD

**Discussion by erjank** [cssi-pi-community/2022-meeting#11] [Panel 1 roundtable (Monday morning)]
Starting the discussion for table 22!

**Comment by erjank** [comment]
How realistic is it to effect cultural changes in departments and academic units through mainly research-focused grants? What funding or reward mechanisms are there for say "inclusivity champions" in compute-centered units?

**Comment by erjank** [comment]
Some initial ost-panel discussion topics: 

* Sustainability: No place to send a grant to maintain things- we always have to over-promise new features.
* Recognition of not enough money right now to pay CI professionals equitably and competitively
* Student recruiting hard: Hard to explain how building workflows is a marketable skill
* Discussion of collaboration with nonprofits for sustainability
* Discussion of need for multiple levels of support for different kinds of projects
*

**Comment by erjank** [comment]
Putting small level funding into maintenance directly. Also need to put support into platforms that make software available.

**Comment by erjank** [comment]
Discussion of postdoc model for working on software temporarily before moving on to industry.

**Comment by erjank** [comment]
Disambiguation of sustaining an organizational home for CI professionals, and separately supporting continuity of knowledge within those organizations.

**Comment by erjank** [comment]
Discussion of institutional reward structure change being a critical component of overall community change

**Reply by erjank** [reply]
There aren't institutional costs to expanding what's being used for faculty evaluation, so including data and software seems feasible.  Wonder if NSF should offer that guidance for how to update those metrics?

**Comment by erjank** [comment]
Importance of dispelling the idea that research software engineers are just debuggers - they're not just writing code, they're thinking creatively and contributing to research enterprise.

**Comment by erjank** [comment]
Discussion of user recruitment and user support. Testbed example, 96% of users struggle to start b/c of CLI barriers. Developers oversubscribed to help users. Need for low-level funding for help all new users onboard. Expanding knowledgebase of folks who know about and use infrastructure.

**Reply by natvilros** [reply]
Very important topic. The idea of "build it and they'll come" does not always work. As described in the morning talks, not only how to support new users but also how to let people be aware of CI that can help them.

**Reply by erjank** [reply]
The Carpentries does well with this, and I see lots of partnerships with them across the CSSI community, and I'd be curious learning what fractions of folks do, and how well it's working for them!

**Reply by erjank** [reply]
https://carpentries.org/

**Comment by erjank** [comment]
Discussion of importance of community development.

**Comment by erjank** [comment]
Real consensus around how we want to have sustained funding for how we're doing, not over-proposing new features.

**Comment by erjank** [comment]
Inefficiency of paying centers to pay people for working on a project versus supporting project directly.

**Comment by erjank** [comment]
Sustainable size of Cyberinfrastructure is much smaller than, say, telescopes, though!

**Comment by erjank** [comment]
Discussion of hiring professional positions in academic professional positions instead of postdocs to improve permanence/stability.

**Comment by erjank** [comment]
Discussion of maternity leave: Allow project reports to be delayed for PIs on family leave.
